frontend/packages/dev-console/integration-tests/features/addFlow/export-application.feature

```diff
@@ -0,0 +1,36 @@
+@add-flow
+Feature: Export of application from the Add page
+              As a user, I have an unmanaged application which I want to export. I'd like to be able to later add that code to git or some shared location so that I can share with others, or import into a new cluster or same cluster but different project, or be able to apply updates to an existing application.
+
+
+
+    #The operator mentioned below needs to be installed beforehand due to the issue in its installation
+        Background:
+             Given user has installed Gitops primer Operator
+              And user is at developer perspective
+              And user has created or selected namespace "aut-export-application"
+
+
+
+        @regression @odc-6738
+        Scenario: Export application option on the Add page: A-14-TC01
+            Given user has created "nodejs-ex-git-1" workload in "nodejs-ex-git-app" application
+             When user navigates to Add page
+              And user clicks on Export Application option
+              And user can see the Export Application modal for the user to start the export
+              And user clicks on Ok button
+             Then user can see a toast message saying "Export of resources in aut-export-application has started."
+              And user can see a toast message saying "All the resources are exported successfully from aut-export-application. Click below to download it." and close it
+              And user navigates to Topology page
+              And user can see primer job created in topology
+        
+        
+        @regression @odc-6296
+        Scenario: Export Application modal when export application is in progress: A-14-TC02
+            Given user is at Add page
+             When user clicks on Export Application option
+              And user can see the Export Application modal for the user to start the export
+              And user clicks on Ok button
```

Comment:
In when statement we put actions performed by the user, can we modify statements as
```suggestion
              And user clicks on Ok button on Export Application modal to start the export
```